kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -75,22 +74,46 @@
         v-model="confirmedPassword"
       />
 
-      <ui-select
-        :name="$tr('selectFacility')"
-        :placeholder="$tr('selectFacility')"
-        :label="$tr('facility')"
-        :value="selectedFacility"
-        :options="facilityList"
-        :invalid="facilityIsInvalid"
-        :error="facilityIsInvalidText"
-        @blur="facilityBlurred = true"
-        @input="updateSelection"
-      />
+      <!-- ToS modal only shows up if the box empty prior to clicking -->
+      <!-- Using UI Checkbox for blur events and v-model -->
+      <label for="terms-agreement-checkbox">
+        <button
+          type="button"
+          class="terms-agreement-view-prompt"
+          @click="showTerms = true"
+        >
+          {{ $tr('viewTermsOfServicePrompt') }}
+        </button>
+
+        <k-checkbox
```

Comment:
Let's wrap this checkbox and the terms-agreement-error-box in a div with a min-height that equals the height of the checkbox and the terms-agreement-error-box so that the submit button does not jump up and down. Also some margin-bottom would be nice.

Reply:
Just used the label, but it's done